build/npm/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+node_modules
+bin
```

Comment:
We need the `bin` folder to exists. Why is it ignored?

Reply:
the `gauge` inside the bin is a placeholder, to prevent `npm install` from erroring out with "FileNotFound". This folder is not supposed to be tracked by vcs.